workspacemanager/MantidWorkspaceProvider.py

```diff
@@ -0,0 +1,17 @@
+from WorkspaceProvider import WorkspaceProvider
+from mantid.simpleapi import mtd,Load,DeleteWorkspace,Load,GroupWorkspaces,RenameWorkspace,SaveNexus
+
+class MantidWorkspaceProvider(WorkspaceProvider):
```

Comment:
This is generally good. Doc strings needed, and casing should match PEP8 standards.